src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './footer.css'
+import { Link } from 'react-router-dom'
+
+export const Footer = () => {
+  return (
+    
+            <div className="footer-container">
+             
+
+                    <p className='footer-title'>Made with ❤️ by Shubham</p>
+                    <p className='footer-title'>Copyright &copy; 2022 Neos-Quiz, INC. All Right Reserved.</p>
+           
+
+                <div className="link-container">
+
+                        <Link to="https://github.com/Shubhamkr2610" target="_blank" className="social-link"><i
```

Comment:
It will not work as you are thinkin😁

It will work like :
With Link component of react-router you can do that. In the "to" prop you can specify 3 types of data:

a string: A string representation of the Link location, created by concatenating the location’s pathname, search, and hash properties.
an object: An object that can have any of the following properties:
pathname: A string representing the path to link to.
search: A string representation of query parameters.
hash: A hash to put in the URL, e.g. #a-hash.
state: State to persist to the location.
a function: A function to which current location is passed as an argument and which should return location representation as a string or as an object
For your example (external link):

https://example.zendesk.com/hc/en-us/articles/123456789-Privacy-Policies

You can do the following:

<Link to={{ pathname: "https://github.com" }} target="_blank" />

Or you can use anchor tag instead

Reply:
Thankyou @arjun-computer-geek , nice pick . when I was building this , it wont worked when I checked . I thought somehow  my link will broken . but don't have any about this to me . Thankyou for sharing this . This helped me alot

Reply:
hey @arjun-computer-geek the link you provided is broken . Can you provide the link again . Please